src/main/java/bob/Ui.java

```diff
@@ -0,0 +1,100 @@
+package bob;
+
+import bob.Task.Task;
+
+import java.util.Scanner;
+
+public class Ui {
+    private final String lineSplit = "====================================================================";
+    private final String logo = "██╗░░██╗██╗  ██╗██╗███╗░░░███╗  ██████╗░░█████╗░██████╗░\n"

```

Comment:
Constant names must be all uppercase using underscore to separate words, i.e. LINE_SPLIT and LOGO

Reply:
Thanks for pointing this out! Will make the changes 👍